servers/partner-ci/builders/common.prepare.sh

```diff
@@ -70,28 +70,30 @@ function smart_erase {
       fi
     done
   fi
-  for vm in $vms; do
-    domstat=$(virsh domstate $vm)
-    if [[ $domstat != *"shut"* ]]; then
-      if virsh destroy $vm; then
-        echo "domain destroyed succesfully"
-        virsh undefine --remove-all-storage --snapshots-metadata $vm
-      else
-        ref=$?
-        echo "there are some troubles with virt stack, restart services and recheck"
-	sudo service libvirt-bin restart
+
+  if [ ! -z "$vms" ]; then
+    for vm in $vms; do
```

Comment:
`if [ ! -z "${vms}" ]; then ... fi `<--remove
 ` for vm in ${vms:?}; `<-- change to